lib/logstash/outputs/elasticsearch.rb

```diff
@@ -250,7 +254,7 @@ def register
       client_settings["network.host"] = @bind_host if @bind_host
       client_settings["transport.tcp.port"] = @bind_port if @bind_port
       client_settings["client.transport.sniff"] = @sniffing
- 
+
```

Comment:
Yeah, I noticed that too and was unhappy. Do I need to back them out? Should be easy to do.